src/main/resources/i18n/messages.properties

```diff
@@ -95,7 +95,6 @@ email.notification.aux.difficulty.hard=Hard
 
 # Plagiarism
 email.notification.title.post.plagiarismVerdict=Verdict reached in plagiarism case for exercise {0}
-email.notification.title.post.plagiarismCase=New plagiarism case in exercise {0}
```

Comment:
This is not used anymore. The one that is specified at "artemisApp.plagiarism.plagiarismCases.notification.title" is used instead.